### ISSUE TITLE
Comparison with Taichi_blend's tina renderer

### ISSUE THREAD

**Issue by shankarsivarajan** [bsavery/BlenderPythonRenderer#30]
There's another [addon](https://github.com/taichi-dev/taichi_blend/) that also has a [Taichi renderer](https://github.com/taichi-dev/taichi_three/). Have you compared it to yours?

**Comment by bsavery** [comment]
Yes.  It appears it just runs some Taichi sims INSIDE of blender's python.  It's not actually rendering Blender Data. 

I.e so it won't actually render a blender scene.

**Comment by shankarsivarajan** [comment]
You're mistaken. The addon does allow you to run taichi simulations on Blender's python and use Blender as merely an interactive 3D display, and that's all I used it for, but it includes a render engine as well.

**Comment by bsavery** [comment]
Can you load an existing blender scene and just render it?  That didn't seem to be the case for me.  If you can prove me wrong I would like to see it!

**Comment by shankarsivarajan** [comment]
It only works with Principled BSDF shader, so I can't do the BMW scene, but here's Suzanne:
![tina_suzanne](https://user-images.githubusercontent.com/16606427/155251433-c5b0d715-551f-4a79-97ef-e8abe8baf524.png)

Incidentally, I had to uninstall the version of taichi I installed through pip before it would work. It comes with its own (outdated) version 0.7.21.

**Comment by bsavery** [comment]
Interesting.  It might interesting as a further project to merge that code to allow faster interactive renders

**Comment by bsavery** [comment]
Thinking a bit more it would be a really cool (and big) project to do a full scalable renderer that could do fast rasterization (like the above) and path tracing for final renders.  And this could turn into the basis for that, but like I said it would be a big project and I have a day job ;)

**Comment by bsavery** [comment]
What do you mean exactly that you're trying to do?

**Comment by shankarsivarajan** [comment]
I have several half-baked ideas regarding NPR stylization.
